src/sentry/static/sentry/app/views/issueList/sortOptions.tsx

```diff
@@ -6,43 +6,28 @@ import DropdownControl, {DropdownItem} from 'app/components/dropdownControl';
 import {t} from 'app/locale';
 import space from 'app/styles/space';
 
-class IssueListSortOptions extends React.PureComponent {
+type Props = {
+  sort?: string;
+  onSelect?: (sort: string) => void;
+};
+
+class IssueListSortOptions extends React.PureComponent<Props> {
```

Comment:
Should we refactor to function component now that there's no state?

Reply:
@matejminar great suggestion

Reply:
Iirc function component !== purecomponent

Reply:
Correct, pure component uses shallow comparison on state+props change and prevents unnecessary re-renders.

I probably wouldn't optimize this prematurely though as rerenders in such a simple component are very cheap and overusing PureComponent can sometimes cause more harm than good.

Unless of course, we had some performance issues within this component in the past, in which case we could even move PureComponent up and make `sortOptions`'s parent `IssueListFilters` a PureComponent to encompass all of its children.

What do you think @billyvg ?

Reply:
I think everything you said is right! Just wanted to point out that they are not equivalent